client.go

```diff
@@ -24,8 +28,26 @@ var userAgent = fmt.Sprintf(
 	Version, runtime.Version(), runtime.GOOS, runtime.GOARCH,
 )
 
-func newClient() *Client {
+func newClient(use_proxy bool) *Client {
 	// Return default HTTP client for now. @todo replace with defined client
+	if use_proxy {
+		// Setup localhost TOR proxy
+	 	torProxyUrl, err := url.Parse("socks5://127.0.0.1:9050") // port 9150 is for Tor Browser
+	 	if err != nil {
+	 		fmt.Println("Unable to parse URL:", err)
```

Comment:
Prefer a `log.Fatalln()` here instead of using os.Exit() in the code. It will call that for you. We also want to write our logs to stderr.